code/pizzabot.js

```diff
@@ -4,4 +4,88 @@ const pepperoni = "Pepperoni Pizza"
 
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+var x = vegetarian
+var y = hawaiian
+var z = pepperoni
+
+alert("Hey! Happy to serve you pizza! On our menu we have " + x + ", " + y + " and " + z + ".");
```

Comment:
You can make this line more readable with using `${}`!
```suggestion
alert("Hey! Happy to serve you pizza! On our menu we have ${x}, ${y} and ${z}.");
```